### PR TITLE
PaymentMethod test cases

### PR DESCRIPTION
**Pull request by enishant**
### - Link added should be a valid URL and accessible throughout the org.
   ### - If the branch name contains hotfix / revert by default the BVT workflow check will pass.

| Test Case Document URL                        |
|-----------------------------------------------|
| Please paste test case document link here.... |

### CODE REVIEW

**Review by garimaprachi — COMMENTED** [review]
LGTM

File: Test/Unit/PaymentMethodTest.php [hunk 1]
```diff
@@ -0,0 +1,504 @@
+<?php
+declare(strict_types=1);
+use PHPUnit\Framework\TestCase;
```
Comment (abdulwahidsharief) [inline Test/Unit/PaymentMethodTest.php:4]: @yashgit891 leave a whiteline space

File: Test/Unit/PaymentMethodTest.php [hunk 2]
```diff
@@ -0,0 +1,504 @@
...
+
+/**
+ * @covers Razorpay\Magento\Model\PaymentMethod
+ */
+class PaymentMethodTest extends TestCase {
```
Comment (abdulwahidsharief) [inline Test/Unit/PaymentMethodTest.php]: flower bracket should start with new line

File: Test/Unit/PaymentMethodTest.php [hunk 3]
```diff
@@ -0,0 +1,504 @@
...
+    public function setUp():void
+    {
+
```
Comment (abdulwahidsharief) [inline Test/Unit/PaymentMethodTest.php]: remove white line

**Review by abdulwahidsharief — APPROVED** [review]
LGTM

**Pull request merged**